crates/ironclaw_gateway/static/styles/surfaces/logs.css

```
@@ -1,138 +1,206 @@
-}
-
-.log-entry.expanded {
-  white-space: normal;
 }
 
 .log-entry.expanded .log-msg {
   white-space: pre-wrap;
-  word-break: break-all;
-  overflow: visible;
-  text-overflow: unset;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)